process-compose.yaml

```diff
@@ -251,6 +251,13 @@ processes:
         path: /healthcheck
       failure_threshold: 100
 
+  permissionless-builder:
+    command: permissionless-builder
+    depends_on:
+      sequencer0:
+        condition: process_healthy
+      da_server:
```

Comment:
Is there a reason this doesn't have the condition?

Reply:
I wasn't sure what it could be, and it wasn't the most urgent issue at this time.